server/src/domain/game.rs

```diff
@@ -61,36 +66,16 @@ mod game_test {
     use crate::domain::game::Game;
 
     fn init_game_obj() -> Game {
-        game::init("Hello World".to_string())
-    }
-    #[test]
-    fn update_name() {
-        let mut game_model = init_game_obj();
-
-        game_model.update_game_name(String::from("Antidote"));
-
-        assert_eq!(game_model.name, "Antidote")
+        game::init_lobby("Hello World".to_string(), "Hank".to_string())
     }
 
     #[test]
     fn update_players() {
         let mut game_model = init_game_obj();
-        game_model.add_player(&"Harribel");
-        game_model.add_player(&"Lisa");
-        game_model.add_player(&"Chen");
-
-        assert_eq!(game_model.num_players(), 3)
-    }
-    #[test]
-    fn update_rounds() {
-        let mut game_model = init_game_obj();
-        game_model.update_round();
-
-        assert_eq!(game_model.round, 1);
+        game_model.add_player("Harribel".to_string());
+        game_model.add_player("Lisa".to_string());
+        game_model.add_player("Chen".to_string());
 
-        game_model.update_round();
-        assert_eq!(game_model.round, 2);
-        game_model.update_round();
-        assert_eq!(game_model.round, 3);
```

Comment:
Sorry, I just deleted stuff that I think is a little out of date with my understanding of things. The concept of a round pertains to an actual in progress game, with should be a lower level state than `Game`.